synapse/rest/key/v2/remote_key_resource.py

```diff
@@ -126,7 +125,7 @@ async def _async_render_POST(self, request: Request) -> None:
 
     async def query_keys(
         self,
-        request: Request,
+        request: SynapseRequest,
```

Comment:
I can't see where we use this `SynapseRequest`?

Reply:
It gets fed into `respond_with_json` which now requires a `SynapseRequest`